pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -30,6 +29,22 @@ import (
 )
 
 var (
+	// these are admission plugins that cannot be applied until after the kubeapiserver starts.
+	// TODO if nothing comes to mind in 3.10, kill this
+	SkipRunLevelZeroPlugins = sets.NewString()
+	// these are admission plugins that cannot be applied until after the openshiftapiserver apiserver starts.
+	SkipRunLevelOnePlugins = sets.NewString(
+		"ProjectRequestLimit",
```

Comment:
shouldn't we make this configurable externally? (master config?)

Reply:
> shouldn't we make this configurable externally? (master config?)

I don't think so.  We know which ones require openshift resources.  We aren't providing generic skipping possbilities.